_drafts/cortex-m-watchpoints.md

```diff
@@ -0,0 +1,1097 @@
+---
```

Comment:
Maybe the dumbest example you are missing is something like:

```
for (int i = 0; i < 1000; i++) {
    // I want to do something when i = 100
}
```

I've seen people click the space bar 100 times. I've been to a conference talk where they did this. Might be worth mentioning directly.